README.md

```diff
@@ -4,6 +4,8 @@ A Visual Studio Code test extension that allows users to run their [Kani Rust Ve
 
 ## Usage
 
+Check [user guide](docs/user-guide.md) for more detailed information.
+
 ![Kani Usage](resources/screenshots/kani-demo.png)
 
 1.  Open a rust crate or workspace in Visual Studio Code
```

Comment:
```suggestion
1.  Open a Rust package in Visual Studio Code.
```

Reply:
Wonder if we should rename the `Kani Proofs` panel to `Kani harnesses`... Let's discuss offline.

Reply:
Also, maybe we can use ▶️ instead of "play button" (or add the emoji next to that) so it's clearer?